estate/views/estate_menus.xml

```diff
@@ -0,0 +1,11 @@
+<odoo>
+    <menuitem id="real_test_app" name="REAL TEST">
+        <menuitem id="second_level_menu" name="Estate">
+            <menuitem id="third_level_menuitem" name="Properties" action="estate_property_window"/>
```

Comment:
Maybe the naming of the id could be more explicit here like "estate_property_menu"